utils/request/index.js

```diff
@@ -0,0 +1,29 @@
+import axios from 'axios';
+import BASE_URL from '../../constants';
+
+const axiosInstance = axios.create({
+  baseURL: BASE_URL,
+  timeout: 1000,
+});
+
+export const makeRequest = (obj) => {
```

Comment:
proper naming obj can be renamed as requestConfig

Reply:
naming requestObject because it will also have other info i.e data, and config is a bit misleading